tests/test_plotting.py

```diff
@@ -35,404 +35,543 @@
     fig, ax = plot_fsa(eq, "B_theta")
     return fig
 
 
+@pytest.mark.unit
+@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
-def test_fsa_G(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
+def test_fsa_G(DSHAPE_current):
+    """Test plotting of flux surface average poloidal current."""
+    eq = EquilibriaFamily.load(load_from=str(DSHAPE_current["desc_h5_path"]))[-1]
     fig, ax = plot_fsa(eq, "B_zeta", log=True)
     ax.set_ylim([1e-1, 1e0])
     return fig
 
 
+@pytest.mark.unit
+@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
-def test_fsa_F_normalized(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
-    fig, ax = plot_fsa(eq, "|F|", log=True, norm_F=True)
-    ax.set_ylim([1e-7, 1e-4])
-    return fig
-
-
-@pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
-def test_fsa_F_normalized_vac(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[1]
+def test_fsa_F_normalized(DSHAPE_current):
+    """Test plotting flux surface average normalized force error on log scale."""
+    eq = EquilibriaFamily.load(load_from=str(DSHAPE_current["desc_h5_path"]))[-1]
     fig, ax = plot_fsa(eq, "|F|", log=True, norm_F=True)
-    ax.set_ylim([1e-6, 2e-4])
+    ax.set_ylim([1e-5, 1e-2])
     return fig
 
 
+@pytest.mark.unit
+@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_2d)
-def test_section_J(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
+def test_section_J(DSHAPE_current):
+    """Test plotting poincare section of radial current."""
+    eq = EquilibriaFamily.load(load_from=str(DSHAPE_current["desc_h5_path"]))[-1]
     fig, ax = plot_section(eq, "J^rho")
     return fig
 
 
+@pytest.mark.unit
+@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=24)
-def test_section_Z(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
+def test_section_Z(DSHAPE_current):
+    """Test plotting poincare section of Z coordinate."""
+    eq = EquilibriaFamily.load(load_from=str(DSHAPE_current["desc_h5_path"]))[-1]
     fig, ax = plot_section(eq, "Z")
     return fig
 
 
+@pytest.mark.unit
+@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_2d)
-def test_section_R(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
+def test_section_R(DSHAPE_current):
+    """Test plotting poincare section of R coordinate."""
+    eq = EquilibriaFamily.load(load_from=str(DSHAPE_current["desc_h5_path"]))[-1]
     fig, ax = plot_section(eq, "R")
     return fig
 
 
+@pytest.mark.unit
+@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_2d)
-def test_section_F(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
+def test_section_F(DSHAPE_current):
```

Comment:
the new figure for this has only 2 orders of magnitude on the colorscale, is the new soln just have a radial force component that spans less orders of magnitude?

Reply:
Yeah it's slightly worse in terms of force error, I reduced the resolution and number of continuation steps a bit to make it faster.